docs/manual/java/guide/build/EclipseMavenNew.md

```diff
@@ -0,0 +1,47 @@
+# Creating a new Maven Eclipse project with the Lagom archetype
+
+The Lagom Maven archetype allows you to quickly create a project for development. Follow these steps to:
+
+[Check Prerequisites](#Check-prerequisites)
+ [Create your project](#Create-an-Eclipse-project)
+
+## Check prerequisites
+
+Before attempting to create a Lagom Maven project in Eclipse, ensure that Eclipse is configured with the following:
+
+* An [m2eclipse](http://www.eclipse.org/m2e/documentation/m2e-documentation.html) plugin compatible with Maven 3.3 or higher.
+* A JDK 1.8 or higher.
+
+## Create an Eclipse project
+
+The screen shots on this page reflect Eclipse Neon (4.6.2) with M2E (Maven Integration for Eclipse) version 1.7.0. If you are using different versions, screens might differ, but the procedures should be the same.
+
+In Eclipse, follow these steps to create a project using the Lagom Maven archetype:
+
+1. From the **File** menu, select **New > Project**.  
+    The **New Project** screen opens. 
+1. Expand **Maven**, select **Maven Project**, and click **Next**. 
+    The **New Maven project** wizard opens. [[EclNewMavenProj.png]]
+1. Leave the default, **Use default Workspace location** box selected and click **Next**. 
+    The **Select an archetype** page opens.[[EclSelectArch.png]]
+1. If the Lagom archetype appears in the list, select it. If not, click **Add Archetype** and supply the following values:
+    * **Archetype Group Id:** com.lightbend.lagom
+    * **Archetype Artifact Id:** maven-archetype-lagom-java
+    * **Version:** The Lagom version number, such as 1.3.0.
+    * **Repository URL:** Leave blank
```

Comment:
I would insist here that the screenshot may be obsolete and not pointing to the latest usable version, or blur the version field in the image to force users to read this info.

We already had bug reports because users would blindly copy paste snippets that we had not maintained to the latest minor (docs said 1.2.0, latest was 1.2.2).